docs/tutorials/multimodal/ner.md

```diff
@@ -0,0 +1,76 @@
+# AutoMM for Named Entity Recognition
+
+Named entity recognition (NER) refers to identifying and categorizing key information (entities) from unstructured text. An entity can be a word or a series of words which correspond to categories such as cities, time expressions, monetary values, facilities, person, organization, etc. An NER model usually takes as input an unannotated block of text and output an annotated block of text that highlights the named entities with predefined categories. For example, given the following sentences, 
+
+- Albert Einstein was born in Germany and is widely acknowledged to be one of the greatest physicists.
+
+The model will tell you that "Albert Einstein" is a PERSON and "Germany" is a LOCATION. In the following, we will introduce how to use AutoMM for the NER task, including how to prepare your data, how to train your model, and what you can expect from the model outputs.
+
+ 
+## Data Preparation
+Like other tasks in AutoMM, all you need to do is to prepare your data as data tables (i.e., dataframes) which contain a text column and an annotation column. The text column stores the raw textual data which contains the entities you want to identify. Correspondingly, the annotation column stores the label information (e.g., the *category* and the *start/end* offset in character level) for the entities. AutoMM requires the annotation column to have the following json format. 
+
+- [{"entity_group": "PERSON", "start": 0, "end": 15}, 
+{"entity_group": "LOCATION", "start": 28, "end": 35}]
+
+where **entity_group** is the category of the entity and **start** is a character position indicating where the entity begins while **end** represents the ending position of the enity. To make sure that AutoMM can recognise your json annotations, it is required to use the exactly same keys/properties (entity_group, start, end) specified above when constructing your data. You can annote "Albert Einstein" as a single entity group or you can also assign each word a label.
+
+If you are already familar with the NER task, you probably have heard about the [BIO](https://en.wikipedia.org/wiki/Inside%E2%80%93outside%E2%80%93beginning_(tagging)) (Benginning-Inside-Outside) format. You can adopt this format (which is not compulsory) to add an *I-prefix* or a *B-prefix* to each tag to indicate wether the tag is the beginning of the annotated chunk or inside the chunk. For example, you can annotated "Albert" as "B-PERSON" because it is the beginning of the name and "Einstein" as "I-PERSON" as it is inside the PERSON chunk. You do not need to worry about the *O* tags, an *O* tag indicates that a word belongs to no chunk, as AutoMM will take care of that automatically. 
```

Comment:
Nit:  `For example, you can annotated "Albert"` --> `For example, you can annotate "Albert"`

Reply:
resolved in [commit](https://github.com/awslabs/autogluon/pull/2232/commits/c71ce038499bfff90493cbb9afac904b2df5ca0a)